src/frontend/src/env/networks.icrc.env.ts

```diff
@@ -385,6 +387,35 @@ const CKXAUT_IC_DATA: IcCkInterface | undefined = nonNullish(CKERC20_PRODUCTION_
 		}
 	: undefined;
 
+/**
+ * Additional ICRC tokens from JSON file
+ */
+const mapIcrcData = (
+	icrcTokens: EnvAdditionalIcrcTokens
+): Record<EnvTokenSymbol, Omit<IcInterface, 'position'>> =>
+	Object.entries(icrcTokens).reduce(
+		(acc, [key, value]) => ({
+			...acc,
+			...((STAGING || BETA || PROD) &&
```

Comment:
**Why only these env vars?**
Because it somehow seems to matter here: https://github.com/dfinity/oisy-wallet/blob/22c0653650a84615e049c0d6ec61f21e45637797/src/frontend/src/env/networks.icrc.env.ts#L225
So i thought it might also matter for this mapping

**Why I only use it for the production tokens?**
I could add a line like this: 
`const ADDITIONAL_ICRC_STAGING_DATA = mapIcrcData(additionalIcrcTokensStaging);`

but because there is no staging token that we want to read right now, this variable will be unused. So i think is better to introduce it when it's needed.

Reply:
ah understood! thank you for the explanation! then, shouldn't the check be only for BETA and PROD?

Reply:
I don't feel strongly about this as I just replicated the existing pattern, i'll remove the staging for now